hilti/toolchain/include/ast/type.h

```diff
@@ -378,6 +378,9 @@ class QualifiedType : public Node {
     /** Returns the type's "sideness". */
     auto side() const { return _side; }
 
+    /** Shortcut to try-cast to `type::Name`. */
+    type::Name* alias() const;
```

Comment:
Returning a pointer to a mutable subverts the method being `const` and we seem to have a number of these since the recent refactor. Could you as a follow-up change this to be

```suggestion
    const type::Name* alias() const;
```

This requires adjusting the signature for `resolvedDeclaration` as well.